typescript/lib/networks/network_context.ts

```diff
@@ -0,0 +1,440 @@
+
+	// The location on the filesystem where this code is running where the enclave data volume is mounted
+	private readonly enclaveDataVolMountpoint: string;
+
+
+    /*
+    Creates a new NetworkContext object with the given parameters.
+    */
+    constructor(
+            client: ApiContainerServiceClient,
+            enclaveDataVolMountpoint: string) {
+                this.client = client;
+                this.enclaveDataVolMountpoint = enclaveDataVolMountpoint;
+    }
+
+    // Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+    public loadLambda(
+            lambdaId: LambdaID,
+            image: string,
+            serializedParams: string): [LambdaContext, Error] {
+        const args: LoadLambdaArgs = newGetLoadLambdaArgs(lambdaId, image, serializedParams);
+        // TODO TODO TODO - CALLBACK & ERROR-HANDLING
+        // // We proxy calls to Lambda modules via the API container, so actually no need to use the response here
+        // _, err := networkCtx.client.LoadLambda(context.Background(), args)
+        // if err != nil {
+        //     return nil, stacktrace.Propagate(err, "An error occurred loading new module '%v' with image '%v' and serialized params '%v'", lambdaId, image, serializedParams)
+        // }
+        const moduleCtx: LambdaContext = new LambdaContext(this.client, lambdaId); //TODO - move to constructor calls?
+        return [moduleCtx, null];
+    }
+
+    // Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+    public getLambdaContext(lambdaId: LambdaID): [LambdaContext, Error] {
+        const args: GetLambdaInfoArgs = newGetLambdaInfoArgs(lambdaId);
+        // TODO TODO TODO - CALLBACK & ERROR-HANDLING
+        // // NOTE: As of 2021-07-18, we actually don't use any of the info that comes back because the LambdaContext doesn't require it!
+        // _, err := networkCtx.client.GetLambdaInfo(context.Background(), args)
+        // if err != nil {
+        //     return nil, stacktrace.Propagate(err, "An error occurred getting info for Lambda '%v'", lambdaId)
+        // }
+        const lambdaCtx: LambdaContext = new LambdaContext(this.client, lambdaId); //TODO - move to constructor calls?
+        return [lambdaCtx, null];
+    }
+
+    // Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
+    public registerStaticFiles(staticFileFilepaths: Map<StaticFileID, string>): Error {
+        const strSet: Map<string, boolean> = new Map();
+        for (let staticFileId in staticFileFilepaths) {
+            const srcAbsFilepath: string = staticFileFilepaths[staticFileId];
+            
+            // Sanity-check that the source filepath exists
+            fs.stat(srcAbsFilepath, (exists) => {
+                if (exists != null) {
+                    return new Error("Source filepath " + srcAbsFilepath + " associated with static file " + staticFileId + " doesn't exist");
```

Comment:
I _think_ that this `return` statement, because it's inside the callback function, will actually just return from teh callback function - not the outer `registerStaticFiles` function